src/local_neighborhood.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import warnings
+from src.PRM import PRM
+from pathlib import Path
+from src.util import prepare_volume, run_container
+
+__all__ = ['LocalNeighborhood']
+
+class LocalNeighborhood(PRM):
+    required_inputs = ['network', 'nodes']
+
+    def generate_inputs(data, filename_map):
+        """
+        Access fields from the dataset and write the required input files
+        @param data: dataset
+        @param filename_map: a dict mapping file types in the required_inputs to the filename for that type
+        @return:
+        """
+        for input_type in LocalNeighborhood.required_inputs:
+            if input_type not in filename_map:
+                raise ValueError(f"{input_type} filename is missing")
+
+        #
+        #
+        #
+        # I am kind of unsure about how to interpret prize set as for local neiborhood
+        if data.contains_node_columns('prize'):
+            #NODEID is always included in the node table
+            node_df = data.request_node_columns(['prize'])
+        elif data.contains_node_columns(['sources','targets']):
+            #If there aren't prizes but are sources and targets, make prizes based on them
+            node_df = data.request_node_columns(['sources','targets'])
+            node_df.loc[node_df['sources']==True, 'prize'] = 1.0
+            node_df.loc[node_df['targets']==True, 'prize'] = 1.0
+        else:
+            raise ValueError("LocalNeighborhood requires nodes and networks")
+
+        #Omics Integrator already gives warnings for strange prize values, so we won't here
+        node_df.to_csv(filename_map['prizes'],sep='\t',index=False,columns=['NODEID','prize'],header=['name','prize'])
+
+        #For now we assume all input networks are undirected until we expand how edge tables work
+        edges_df = data.get_interactome()
+        edges_df['directionality'] = 'U'
+        edges_df.to_csv(filename_map['edges'],sep='\t',index=False,columns=['Interactor1','Interactor2','Weight','directionality'],header=['protein1','protein2','weight','directionality'])
+
+
+    # TODO: implement run function
+    @staticmethod
+    def run(nodetypes=None, network=None, output_file=None, k=None, singularity=False):
```

Comment:
```suggestion
    def run(nodes=None, network=None, output_file=None, k=None, singularity=False):
```
This will need to match `nodes` that was listed in `required_inputs` above.